assets/queries/cloudFormation/s3_bucket_versioning_not_enabled/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+CxPolicy [ result ] {
+
+  resource := input.document[i].Resources[name]
+
```

Comment:
Add a check to see if the resource is a S3 Bucket.
resource.Type == "AWS::S3::Bucket"